src/main/webapp/app/shared/metis/posting-content/posting-content-part/posting-content-part.component.scss

```diff
@@ -1,4 +1,4 @@
-@import 'src/main/webapp/content/scss/artemis-variables';
+@import 'src/main/webapp/app/shared/metis/metis.component';
```

Comment:
[offtopic] Was the `artemis-variables` import causing some issues with the input fields turning grey?